source/SoundManager/Sounds.cs

```diff
@@ -364,27 +364,26 @@ public SoundSource PlaySound(SoundBuffer buffer, double pitch, double volume, Op
 			{
 				Array.Resize(ref Sources, Sources.Length << 1);
 			}
-			Sources[SourceCount] = new SoundSource(buffer, buffer.Radius, pitch, volume, position, parent, 0, looped);
+			Sources[SourceCount] = new SoundSource(buffer, buffer.Radius, pitch, volume, position, parent, looped);
 			SourceCount++;
 			return Sources[SourceCount - 1];
 		}
-
+		
 		/// <summary>Plays a sound.</summary>
 		/// <param name="buffer">The sound buffer.</param>
 		/// <param name="pitch">The pitch change factor.</param>
 		/// <param name="volume">The volume change factor.</param>
-		/// <param name="position">The position. If a train and car are specified, the position is relative to the car, otherwise absolute.</param>
-		/// <param name="train">The train the sound is attached to, or a null reference.</param>
-		/// <param name="car">The car in the train the sound is attached to.</param>
+		/// <param name="position">The position. If a train car is specified, the position is relative to the car, otherwise absolute.</param>
+		/// <param name="car">The parent object the sound is attached to, or a null reference.</param>
 		/// <param name="looped">Whether to play the sound in a loop.</param>
 		/// <returns>The sound source.</returns>
-		public SoundSource PlaySound(SoundBuffer buffer, double pitch, double volume, OpenBveApi.Math.Vector3 position, object train, int car, bool looped)
+		public SoundSource PlaySound(SoundBuffer buffer, double pitch, double volume, OpenBveApi.Math.Vector3 position, AbstractCar car, bool looped)
```

Comment:
```csharp
public SoundSource PlaySound (SoundBuffer buffer, double pitch, double volume, OpenBveApi.Math.Vector3 position, Abstract Car car, bool looped)
```
I think that this overload is unnecessary.

```csharp
public SoundSource PlaySound(SoundBuffer buffer, double pitch, double volume, OpenBveApi.Math.Vector3 position, object parent, bool looped)
```
You can make it common with this function.

Reply:
Absolutely, thanks :)